### ISSUE TITLE
Is this correct behavior?

### ISSUE THREAD

**Issue by Night1099** [giuvecchio/matfuse-sd#9]
Im on Epoch 11 now of a 130000 dataset, all validation hasnt changed from first epochs, but loss is still going down

reconstruction
![reconstruction_gs-068749_e-000010_b-000006](https://github.com/giuvecchio/matfuse-sd/assets/90132896/e5ce4371-d852-4f39-8754-1c27544feb75)

samples quantized

![samples_x0_quantized_gs-068749_e-000010_b-000007](https://github.com/giuvecchio/matfuse-sd/assets/90132896/ef3ba541-c3cc-4837-9612-a2ff4a4079ae)

i am of course fine to wait longer for training but is this correct?

**Comment by giuvecchio** [comment]
Hi, it doesn't look like the correct behavior. Which part of the model are you training? Is it the autoencoder or the U-Net?

**Comment by Night1099** [comment]
The LDM step

The autoencoder came out quite well

**Comment by Night1099** [comment]
i also found this odd from step 1 of LDM training the loss was about .01 and now is down to .0009 ish but with above results

**Comment by giuvecchio** [comment]
Looking at the reconstruction, it seems that the autoencoder weights are not loaded correctly. Do the console logs look correct when starting or resuming the training?

**Comment by Night1099** [comment]
here is full train start log

https://pastecode.io/s/jd5h5pw1

Validation sanity does come up and says 0/2 even tho i know when training autoencoder it said 0/63

Would that be the problem?

and you can see that it sees 63

```
#### Data #####
train, MatFuseDataset, 173319
validation, MatFuseDataset, 63
```

**Comment by Night1099** [comment]
ah dang it says missing keys for all those i thought that was found keys

i assume thats the problem haha

its just the raw ckpt file the autoencoder train step gave out

**Comment by giuvecchio** [comment]
Yes, that's the issue! Looks like the checkpoint loading is not correctly looking for the weights under "state_dict". Fixing it right away! In the meanwhile you can look at the `init_form_checkpoint` method in the base VQModel

**Comment by giuvecchio** [comment]
Ok, should be fixed in f0e773c!

**Comment by Night1099** [comment]
ok it started without any key readout, ill let this run and get back to you 



Thank you!

**Comment by Night1099** [comment]
ok training looks correct now thanks again
